lib/tasks/evm_dba.rake

```diff
@@ -55,6 +55,11 @@ namespace :evm do
       Rake::Task['db:seed'].invoke
     end
 
+    desc "Run the database failover monitor"
+    task :run_failover_monitor => :environment do
+      EvmDatabase.run_failover_monitor
```

Comment:
As mentioned in https://github.com/ManageIQ/manageiq-postgres_ha_admin/pull/10, I don't like a rake task if there's a remote chance we'd need to pass args to the monitor script.  I believe the ease of loading environment is the same as putting `require 'config/environment'` or something to what we do in our tools directory scripts.

I don't know that it makes sense to be a rake task.  Should this be exposed, by specifying a `desc` ?

Reply:
Right, so my concern was that I didn't want to have to specify the whole path to `/var/www/miq/vmdb/config/environment` in some ruby script (in the manageiq-appliance repo). So this seemed like a cleaner solution.

Just to be clear I think the alternative would be a new file in manageiq-appliance which would do something like this:

```ruby
require '/var/www/miq/vmdb/config/environment'
EvmDatabase.run_failover_monitor
```

I may be missing a better solution though ...


> I don't know that it makes sense to be a rake task. Should this be exposed, by specifying a desc ?

Maybe not? I ran it manually a few times to test, but I feel like this logic ("this shouldn't be runnable directly by users") precludes it from being a `tool` as well.

Reply:
I guess we could create a APP_ROOT or something in appliance so you don't have to know the root of the application, something like APPLIANCE_ROOT for the checkout of the appliance repo: https://github.com/ManageIQ/manageiq-appliance/blob/master/LINK/etc/default/evm#L15

Reply:
> Maybe not? I ran it manually a few times to test, but I feel like this logic ("this shouldn't be runnable directly by users") precludes it from being a tool as well.

Ok, so maybe we can drop the description before we merge.